main.js

```diff
@@ -297,35 +505,9 @@ function cacheIt(text, language) {
     });
 }
 
-function sayIt(text, language, volume, processing) {
+async function sayIt(text, language, volume, processing) {
     let md5filename;
 
-    // Extract language from "en;volume;Text to say"
```

Comment:
There is a seperate datapoint for volume. The language option is useless, while it was already broken.

Reply:
But setting volume in a separate state is not working "atomic" because you have no control over it and more complicated for users ... I think they use that often. And for language: Why not fixing it in this PR?

Reply:
The language feature makes no sense, the engine and the language are predefined in the settings. Depending on the engine you have to choose more than only the language, like the voice. 
The proper way to use more than one language is to setup new instances.

Reply:
> because you have no control over it

What do you mean with this statement? You have always the control, set it before the text and go. The other thing is, normal this ist once and never Touched again.

Reply:
Ok, I'm not using it so I can not really say about importance. But then the changelog should directly have info that this is not allowed anymore like 

When sending text it is no longer allowed to specify the language and volume (e.g. "en;volume;text" is no longer working)

That it is really clear for the users .. I would not have guessed from the existig changelog entries that this is that way.

Reply:
AAAAND: https://github.com/ioBroker/ioBroker.sayit/blob/master/admin/blockly.js also need to be adjusted because it uses it

Reply:
> What do you mean with this statement? You have always the control, set it before the text and go. The other thing is, normal this is set once and never Touched again.

setState is async ... so if I fire two setState after each other there is a chance that volume was not already set before the other setState ... when same script is doing it then it is "Mostly working" but when another script changes the volume in the mentime and "milliseconds between the set-volume and set-text" ... enough cases possible